interpreter.py

```diff
@@ -1,46 +1,68 @@
 from __future__ import with_statement
-
-import sys
-
 from rpython.rlib import jit
 from llvm_wrapper import *
+from type_wrapper import *
```

Comment:
I know it shouldn't be used, but I actually use everything from those particular modules. I will change it though.